documents/FAQ.md

```diff
@@ -32,10 +32,10 @@ To enable logging in the samples, you will need to set the following system prop
 * `aws.crt.log.level`: The level of logging shown. Can be `Trace`, `Debug`, `Info`, `Warn`, `Error`, `Fatal`, or `None`. Defaults to `Warn`.
 * `aws.crt.log.filename`: The path to save the log file. Only needed if `aws.crt.log.destination` is set to `File`.
 
-For example, to run `BasicPubSub` with logging you could use the following:
+For example, to run `Mqtt X509` with logging you could use the following:
 
 ```sh
-mvn compile exec:java -pl samples/Mqtt/Mqtt5X509 -Daws.crt.debugnative=true -Daws.crt.log.level=Debug -Daws.crt.log.destionation=Stdout -Dexec.mainClass=pubsub.PubSub -Dexec.args='--endpoint <endpoint> --cert <path to cert> --key <path to key>'
+mvn compile exec:java -pl samples/Mqtt/Mqtt5X509 -Daws.crt.debugnative=true -Daws.crt.log.level=Debug -Daws.crt.log.destionation=Stdout -Dexec.mainClass=mqtt5x509.Mqtt5X509 -Dexec.args='--endpoint <endpoint> --cert <path to cert> --key <path to key>'
```

Comment:
The new samples have been adjusted to no longer need `-Dexec.mainClass=mqtt5x509.Mqtt5X509`. This example command should be simplified to: 
mvn compile exec:java \
    -pl samples/Mqtt/Mqtt5X509 \
    -Dexec.args=" \
    --endpoint <endpoint> \
    --cert <Path to certificate file> \
    --key <Path to private key file>"